dags/longitudinal.py

```diff
@@ -77,7 +77,7 @@
                       email=["telemetry-alerts@mozilla.com", "chudson@mozilla.com"],
                       execution_timeout=timedelta(hours=10),
                       release_label="emr-5.0.0",
-                      instance_count=5,
+                      instance_count=30,
```

Comment:
I thought this wasn't needed any longer after stopping the SparkContext. Did I misunderstand?